go.mod

```diff
@@ -7,8 +7,9 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/deckarep/golang-set v1.7.1
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/golang/protobuf v1.3.1
+	github.com/golang/protobuf v1.5.0
```

Comment:
Looks the latest golang/protobuf version is v1.5.2 based on https://github.com/golang/protobuf/releases. Shall we use the latest version here?